doc/user/content/install.md

```diff
@@ -70,6 +70,16 @@ sudo apt update
 sudo apt install materialized
 ```
 
+### apt DEB822 (Ubuntu 16.04+, Debian 9+)
+
+```shell
+# Add and update the repository
+curl https://raw.githubusercontent.com/MaterializeInc/materialize/main/misc/deb822/materialize.sources | sudo tee /etc/apt/sources.list.d/materialize.sources
```

Comment:
Can you plunk the deb822 file in `misc/www/apt/materialize.sources`? Then you'll be able to curl `https://dev.materialize.com/apt/materialize.sources`, which I think reads quite a bit nicer.

Reply:
Neat!